packages/moonstone/DayPicker/DayPicker.js

```diff
@@ -198,11 +233,37 @@ const DayPickerBase = class extends React.Component {
 		}
 	}
 
+	/**
+	 * Determines whether it should return ariaLabel passed in prop.
+	 *
+	 * @param {Number} selected day type
+	 * @param {String[]} selected day aria labels
```

Comment:
I think this explanation should be `* @param {String[]} Array of day aria labels`.
It's not selected.